settings.gradle

```diff
@@ -0,0 +1,10 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+rootProject.name = "SurveySdk"
+include ':lib'
```

Comment:
I'd rather make a better name here instead of `:lib` 😓  let's say later we will have another `:authentication` lib, it will be confused right?

Reply:
@sleepylee 
You're right, so I will change `:lib` to `:surveylib`, is it ok?

Reply:
I think `survey-sdk` would work; calling just `lib` is a bit vague imo 😅

Reply:
later we can have `survey-auth-sdk`, `survey-filter-sdk`...